target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-map-ac1300.dts

```diff
@@ -0,0 +1,376 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "ASUS Lyra Mini";
+	compatible = "asus,map-ac1300";
+
+	memory {
+		device_type = "memory";
+		reg = <0x80000000 0x8000000>;
+	};
+
+	aliases {
+		led-boot = &led_blue0;
+		led-failsafe = &led_red0;
+		led-running = &led_blue0;
+		led-upgrade = &led_red0;
+	};
+
+	soc {
+		rng@22000 {
```

Comment:
Use existing labels for everything that has one, you should only be left with tcsr nodes here.

Reply:
I'm not sure if I know how to do it. As this is copied from https://git.openwrt.org/?p=openwrt/openwrt.git;a=blob;f=target/linux/ipq40xx/files-4.14/arch/arm/boot/dts/qcom-ipq4019-map-ac2200.dts;h=ea39428b88c80d8760e75b7d63ec298b9e484cb4;hb=9ad3967f140bd6d08796f1ec151b753186b3eb44

Whole work is basically copy paste, but I try to find other supported devices that uses this led driver.

Reply:
This has nothing to do with the LED driver but with the fact that I am constantly seeing just copy/paste that keeps on not using labels in the DTS for these nodes as there is no reason to keep manually overriding the nodes and using the existing label as for wifi etc.

So, instead of doing this hierarchy dance for `rng@22000` you can just use `&prng` in the same way as the UART nodes are.

Reply:
Oh, so this goes one for every xxx@yyyyy should have altered with &zzz to make it right?

As you can probably see I'm not that experienced in programming, so appreciate your understanding.

Reply:
Yes, every node that has labels in the SoC DTSI should use that.

Look at:
https://github.com/openwrt/openwrt/blob/main/target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4029-ws-ap391x.dts